scripts/docker/keycloak/master-realm.json

```diff
@@ -419,7 +420,7 @@
     "requiredActions" : [ ],
     "realmRoles" : [ "default-roles-master", "admin" ],
     "notBefore" : 0,
-    "groups" : [ ]
+    "groups" : [ "/SUPERUSERS" ]
```

Comment:
Is this required? There is already the ort-admin user which is in the superuser group and which is supposed to be used for testing. On the other hand, I guess it doesn't hurt if admin in this group as well.

Reply:
It's a little bit less friction via the UI, Keycloak and the UI share the session. If you want to change something in Keycloak and see its effect in the UI, you need to change the users in between if using the same browser.

So not required, but if there are no strong preferences from you, I'd like to keep it.

Reply:
Thanks, that makes sense.